### ISSUE TITLE
Unknown argument error for plugins

### ISSUE THREAD

**Issue by loremaps** [graphql-cli/graphql-cli#60]
I just tried to install graphql-cli with a plugin (codegen but it is the same with voyager)
```
npm i -g graphql-cli graphql-cli-codegen
```
`graphql -v` reports `2.0.9` and seems to works ok. However, I cannot call the plugin:
`graphql codegen` shows the error:
```
Unknown argument: codegen
```
Am I missing something here? Thanks

**Comment by schickling** [comment]
Is anybody else experiencing this problem?

@loremaps would be good to know more about your setup: OS, Node version, NPM version

**Comment by loremaps** [comment]
OS: Ubuntu 16.04 LTS
npm: 5.5.1
node: 8.9.3

I also reproduced it in another machine with

OS: Ubuntu 16.04 LTS
npm: 5.6.0
node: 8.9.3

**Comment by loremaps** [comment]
It seems that the problem is caused by the 
[jonschlijonschlinkert/npm-paths](https://github.com/jonschlinkert/npm-paths) dependency; which does not return the correct paths for my case. If I hardcode my global npm path here:
https://github.com/graphql-cli/graphql-cli/blob/be73eec100749f07dcfe2ad978ee5dcea195b1fc/src/index.ts#L29

The error is no longer reported.

Specifically, the first path returned by the `npmPaths()` is:
`/home/username/tests/graphql-cli/~/.npm-global/lib/node_modules`
but in my case the correct path is:
`/home/username/.npm-global/lib/node_modules`

**Comment by schickling** [comment]
Thanks for further digging into this @loremaps. Would be great if you can figure out a solution that works also in your case that we can introduce as a fix in `graphql-cli`.

**Comment by loremaps** [comment]
Fixed in the latest release `2.1.3`. Thanks @schickling
